static/src/javascripts/bower.json

```diff
@@ -6,7 +6,7 @@
     "domready": "ded/domready#v0.3.0",
     "eventEmitter": "~4.2.6",
     "fastdom": "0.8.5",
-    "lodash-amd": "~2.4.1",
+    "lodash-amd": "regiskuckaertz/lodash-amd#2.4.2",
```

Comment:
I like approach nb 2, however lodash 4 is not compatible with IE9- so it won't be possible until we upgrade our cut the mustard test :-/